web/src/app/schedules/ScheduleForm.js

```diff
@@ -4,8 +4,8 @@ import { FormContainer, FormField } from '../forms'
 import { TextField, Grid } from '@material-ui/core'
 import { TimeZoneSelect } from '../selection'
 
-export default class ScheduleForm extends React.PureComponent {
-  static propTypes = {
+export default function ScheduleForm(props) {
+  ScheduleForm.propTypes = {
```

Comment:
PR looks great, just one small change to move the propTypes after the component function definition. See an example here: https://github.com/target/goalert/blob/master/web/src/app/rotations/UserForm.js#L27

Reply:
Thank you